near-sdk/tests/store_performance_tests.rs

```diff
@@ -25,6 +25,12 @@ pub enum Collection {
     LookupSet,
     TreeMap,
     Vector,
+    LazyOption,
```

Comment:
I would remove that and instead adjust `perform_asserts` to `fn perform_asserts(total_gas: u64, col: impl Display)` as it's used only for logging purposes. 
It would remove this _ => (col, 0) from other tests. And you can provide your own name for Lazy tests

Reply:
It won't really help the `_ => (col, 0)`, because we have other tests that only test a subset of collections, so I'm not sure it's worth the hassle.